tests/upgrades/test_host.py

```diff
@@ -50,13 +50,13 @@ class TestScenarioPositiveGCEHostComputeResource:
     """
 
     @pytest.fixture(scope='class')
-    def arch_os_domain(self, default_sat):
-        arch = default_sat.api.Architecture().search(query={'search': 'name=x86_64'})[0]
-        os = default_sat.api.OperatingSystem().search(
-            query={'search': 'family=Redhat and major=7'}
-        )[0]
+    def arch_os_domain(self, target_sat):
```

Comment:
Should be `class_target_sat` here. Did `--setup-plan` not catch this??

Reply:
Commented above, the github workflow isn't hitting `tests/upgrades`, an update there should catch it.